synapse/tests/test_tools_backup.py

```diff
@@ -0,0 +1,47 @@
+import os
+import synapse.lib.scope as s_scope
+
+import synapse.tests.utils as s_test
+
+import synapse.tools.backup as s_backup
+
+class BackupTest(s_test.SynTest):
+
+    def dirset(self, dir, skipfns):
```

Comment:
Should we avoid using builtin as var name?

Reply:
Yes